README.md

```diff
@@ -6,6 +6,71 @@ This project contains:
  - `cics-bundle-maven-plugin`, a Maven plugin that builds CICS bundles, including Java-based dependencies.
  - `cics-bundle-reactor-archetype`, a Maven archetype that shows how a simple reactor build can pull together a CICS bundle and a Java project.
 
+## Using `cics-bundle-maven-plugin`
+
+The `cics-bundle-maven-plugin` contributes a new packaging type called `cics-bundle`. This is bound to a new `build` goal that will use the information in the `pom.xml` and dependencies to create a CICS bundle, ready to be stored in an artifact repository or installed into CICS.
+
+To use the `cics-bundle-maven-plugin`:
+
+1. Create a new Maven module for the CICS bundle.
+1. Register the plugin to the `pom.xml` of the CICS bundle module (NB it's not currently available in Maven Central so you'll need to download and [build it](#building-the-project):
+    ```
+    <build>
+      <plugins>
+        <plugin>
+          <groupId>com.ibm.cics</groupId>
+          <artifactId>cics-bundle-maven-plugin</artifactId>
+          <version>0.0.1-SNAPSHOT</version>
+          <extensions>true</extensions>
```

Comment:
Maybe we should explain that you need to run as an extension to get support for the custom packaging type.